roles/reproducer/tasks/configure_crc.yml

```diff
@@ -75,5 +75,13 @@
       local=/testing/
       domain=crc.testing
 
+- name: Configure DNS in CRC VM
```

Comment:
after testing more extensively, this does not seem the right place, the networking info is not pushed to the controller after the `pre-ci-play.yml` file https://github.com/openstack-k8s-operators/ci-framework/blob/415def67377a6e816117046a9b95766aab2467e8/roles/reproducer/files/pre-ci-play.yml#L74 which runs after this